server/src/main/java/org/apache/iotdb/db/localconfignode/LocalConfigNode.java

```diff
@@ -539,7 +539,17 @@ public List<IStorageGroupMNode> getAllStorageGroupNodes() {
    */
   public SchemaRegionId getBelongedSchemaRegionId(PartialPath path) throws MetadataException {
     PartialPath storageGroup = storageGroupSchemaManager.getBelongedStorageGroup(path);
-    return schemaPartitionTable.getSchemaRegionId(storageGroup, path);
+    SchemaRegionId schemaRegionId = schemaPartitionTable.getSchemaRegionId(storageGroup, path);
+    if (schemaRegionId == null) {
+      throw new MetadataException(
+          String.format("Storage group %s has not been prepared well.", storageGroup));
```

Comment:
Print the schema region id.
SchemaRegion-x of Storage group %s is not registered？

Reply:
Already add comments in code to explain this check.